gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.8.3-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.0.2-all.zip
```

Comment:
bin?

Reply:
I think we should stick to using `all`, the difference is that `all` includes source, documentation and examples. The size difference is not that big. 🤷‍♂️

Reply:
🇫🇮